src/test/java/org/beanpod/switchboard/controller/EncoderControllerTest.java

```diff
@@ -47,6 +49,7 @@
   @Mock private EncoderService encoderService;
   @Mock private EncoderMapper encoderMapper;
   @Mock private StreamMapper streamMapper;
+  @Mock private MaintainDeviceStatus maintainDeviceStatus;
```

Comment:
Why import and add a maintainDeviceStatus field here if it is unused?

Reply:
It's being used right now.